syntaxes/flexlm-lic.tmLanguage.json

```diff
@@ -34,13 +34,13 @@
 		"parameters": {
 			"patterns": [{
 				"name": "variable.parameter.flexlm-lic",
-				"match": "\\b(BORROW|DUP_GROUP|FLOAT_OK|HOST_BASED|ISSUED|ISSUER|LINGER|NOTICE|OVERDRAFT|PLATFORMS|SN|START|SUITE_DUP_GROUP|SUPERSEDE|TS_OK|USER_BASED|VENDOR_STRING|HOSTID|SIGN)\\b"
+				"match": "\\b(BORROW|DUP_GROUP|FLOAT_OK|HOST_BASED|ISSUED|ISSUER|LINGER|NOTICE|OVERDRAFT|PLATFORMS|SN|START|SUITE_DUP_GROUP|SUPERSEDE|TS_OK|USER_BASED|VENDOR_STRING|HOSTID|SIGN|COMPONENTS)\\b"
 			}]
 		},
 		"constants": {
 			"patterns": [{
 				"name": "constant.language.flexlm-lic",
-				"match": "\\b(permanent)\\b"
+				"match": "\\b(permanent|\\d{2}-\\w{3}-\\d{4})\\b"
```

Comment:
We often see the line like this where the day part is a single digit:
```
INCREMENT FEATURE_NAME daemonname 2020.07 8-jul-2020 1 \
    (line continues...)
```

Can you support the above pattern?

Reply:
Updated to allow for single digit date